.storybook/preview.js

```diff
@@ -34,7 +34,7 @@ export const preview = {
         ]
       }
     },
-    actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
We didn't use the ability to implicitly add jest spies and [its no longer supported in v8](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#implicit-actions-can-not-be-used-during-rendering-for-example-in-the-play-function), and I ran our stories through the suggested script `yarn dlx storybook migrate find-implicit-spies` against the codebase to make sure.